client/src_editor/Editor_Blocks.re

```diff
@@ -84,8 +91,12 @@ let make =
       _children,
     ) => {
   ...component,
+  retainedProps: {
+    lang: lang,
```

Comment:
retainedProps is deprecated. You need to copy props to state instead

Reply:
Ah, interesting. Ok.